lang/llvm-5.0/Portfile

```diff
@@ -381,10 +396,6 @@ platform darwin {
         configure.args-append -DCOMPILER_RT_BUILD_SANITIZERS=OFF
     }
 
-    if {${subport} eq "clang-${llvm_version}" && ${os.major} <= 9} {
```

Comment:
This change was reverted in the subsequent commit.  Please undo the removal in this commit rather than that one.  I suspect you squashed that change into the wrong commit.

Reply:
I tried something new - reordering commits - and it didn't quite turn out the way I wanted. Let's see if I can bend git to my will...